design-system/react/src/components/Copyable/Copyable.stories.tsx

```diff
@@ -8,8 +11,18 @@ export default {
   },
 };
 
-export const Usage = () => <Copyable value="Some value">Some value</Copyable>;
+export const Usage = () => (
+  <Flex direction="column">
+    <Copyable value="You copied this text">Standard Icon</Copyable>
+    <Copyable
+      value="Another text which you copied"
+      iconProps={{ icon: Icon.is('Airplane'), 'aria-label': 'An Airplane' }}
+    >
+      Different Icon
+    </Copyable>
+  </Flex>
+);
```

Comment:
intead of doing different version in same `Usage` component, it's better to do other components, for example:

```
export const Usage = () => <Copyable value="You copied this text">Standard Icon</Copyable>;
export const UsageIconProps = () => (
  <Copyable
    value="Another text which you copied"
    iconProps={{ icon: Icon.is('Airplane'), 'aria-label': 'An Airplane' }}
  >
    Different Icon
  </Copyable>
)
```

Reply:
@LuizAsFight first of all, thank you for your detailed help,

i made the change that you requested, but the check workflow above still got one error, is this acceptable ?

thanks again.